packages/lodestar-cli/src/cmds/beacon/handler.ts

```diff
@@ -28,8 +29,8 @@ export async function beaconHandler(options: IBeaconArgs & IGlobalArgs): Promise
 
   options = mergeConfigOptions(options);
   const peerId = await readPeerId(beaconPaths.peerIdFile);
-  // read local enr from disk
-  const enr = await readEnr(beaconPaths.enrFile);
+  // read local enr from disk; returns a FileENR which persists enr updates to a file
+  const enr = FileENR.initFromFile(beaconPaths.enrFile, peerId);
```

Comment:
If this PR is putting `FileENR` to use, it should also remove the "save ENR on shutdown" logic.